sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -32,6 +32,7 @@
 
 class Configuration {
 
+    final static String PARTITION_KEY = "/pk";
```

Comment:
this is the default partition-key if the collection does not exist. But if the collection exist the partition-key path could be different and that's ok. right?

So could you please change the naming to `DEFAULT_PARTITION_KEY_PATH`

Reply:
done